src/main/java/ch/njol/skript/expressions/ExprScriptsOld.java

```diff
@@ -49,8 +49,6 @@ public class ExprScriptsOld extends SimpleExpression<String> {
 
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
-		if (this.getParser().hasExperiment(Feature.SCRIPT_REFLECTION))
```

Comment:
The change here appears to be different. This syntax is disabled if the experiment is ***enabled***